src/py42/sdk/queries/fileevents/filters/event_filter.py

```diff
@@ -46,4 +86,43 @@ class InsertionTimestamp(QueryFilterTimestampField):
 
 
 class Source(FileEventFilterStringField):
+    """Class that filters events by event source.
+
+    Available event types are provided as class attributes:
```

Comment:
The are not event types, should be something like `source types`.

Reply:
Changed.